platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/util/BootstrapUtils.java

```diff
@@ -42,19 +130,129 @@
 public final class BootstrapUtils {
 
     /** The logger for this class */
-    private static final Logger logger = LogManager.getLogger(BootstrapUtils.class);
+    private static final Logger LOG = LogManager.getLogger(BootstrapUtils.class);
```

Comment:
I assume `logger` is the pattern that we use now. There was a discussion some time ago about that - but I still write LOG often enough ;)

Reply:
I remember there was a discussion, but I can't remember the outcome. I will change it to logger